### PR TITLE
Aws cloudfront integration

### PR DESCRIPTION
**Pull request by YANG-DB**
### Description
Create an aws cloud-front integration

### Issues Resolved
 - https://github.com/opensearch-project/dashboards-observability/issues/775

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass, including unit test, integration test and doctest
- [x] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by YANG-DB** [comment]
https://github.com/opensearch-project/dashboards-observability/assets/48943349/1b94df4b-59fe-4604-ba7e-5ba799daf817

File: server/adaptors/integrations/__data__/repository/aws_cloudfront/assets/aws_cloudfront-1.0.0.ndjson [hunk 1]
```diff
@@ -0,0 +1,31 @@
...dFromDocValues\":true},{\"count\":0,\"name\":\"aws.cloudfront.x-forwarded-for\",\"type\":\"string\",\"esTypes\":[\"text\"],\"scripted\":false,\"searchable\":true,\"aggregatable\":false,\"readFromDocValues\":false},{\"count\":0,\"name\":\"aws.cloudfront.x-forwarded-for.keyword\",\"type\":\"string\",\"esTypes\":[\"keyword\"],\"scripted\":false,\"searchable\":true,\"aggregatable\":true,\"readFromDocValues\":true,\"subType\":{\"multi\":{\"parent\":\"aws.cloudfront.x-forwarded-for\"}}},{\"count\":0,\"name\":\"aws.cloudfront.x-host-header\",\"type\":\"string\",\"esTypes\":[\"text\"],\"scripted\":false,\"searchable\":true,\"aggregatable\":false,\"readFromDocValues\":false},{\"count\":0,\"name\":\"aws.cloudfront.x-host-header.keyword\",\"type\":\"string\",\"esTypes\":[\"keyword\"],\"scripted\":false,\"searchable\":true,\"aggregatable\":true,\"readFromDocValues\":true,\"subType\":{\"multi\":{\"parent\":\"aws.cloudfront.x-host-header\"}}}]","timeFieldName":"@timestamp","title":"logs-aws-cloudfront-*"},"id":"03112a2b-2e5e-4ab2-bbf5-8cb35582708d","migrationVersion":{"index-pattern":"7.6.0"},"references":[],"type":"index-pattern","updated_at":"2022-03-07T05:46:59.035Z","version":"WzIyOTk0LDFd"}
```
Comment (derek-ho) [inline server/adaptors/integrations/__data__/repository/aws_cloudfront/assets/aws_cloudfront-1.0.0.ndjson:1]: What is up with the no data to display in the video? Is it keyword issue again, can we look into that @YANG-DB

**Review by derek-ho — CHANGES_REQUESTED** [review]
Requesting changes for no data panels in the dashboard

**Comment by derek-ho** [comment]
Still seeing errors and no data panels

File: server/adaptors/integrations/__data__/repository/aws_cloudfront/aws_cloudfront-1.0.0.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
+{
+  "name": "aws_cloudfront",
+  "version": "1.0.0",
+  "displayName": "AWS cloudfront ",
+  "description": "AWS cloudfront Object Store",
+  "license": "Apache-2.0",
+  "type": "logs",
```
Comment (Swiddis) [inline server/adaptors/integrations/__data__/repository/aws_cloudfront/aws_cloudfront-1.0.0.json:7]: Errors on panels caused by type mismatch again

**Review by Swiddis — CHANGES_REQUESTED** [review]

**Review by anirudha — APPROVED** [review]

**Review by Swiddis — APPROVED** [review]
Verified working in manual testing -- only complaints would be to find a better photo of the logo and remove redundant sample data files

**Review by derek-ho — APPROVED** [review]

**Pull request merged**